.github/workflows/continuous_integration.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["pypy-3.9", "3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
+        python-version: ["pypy-3.9", "3.8", "3.9", "3.10", "3.11", "3.12", "3.13", "3.14.0-rc.3"]
```

Comment:
you need to update tox and Makefiles as well to actually be able to build these.

also I think for setup-python we may be able to use 3.14-dev: https://github.com/actions/setup-python